mlir/Makefile

```
@@ -61,6 +63,20 @@ mhlo:
 
 	cmake --build $(MHLO_BUILD_DIR) --target check-mlir-hlo
 
+.PHONY: enzyme
+mhlo:
+	@echo "build enzyme"
+	cmake -G Ninja -S enzyme -B $(ENZYME_BUILD_DIR) \
```

I think this needs to be
```suggestion
	cmake -G Ninja -S Enzyme/enzyme -B $(ENZYME_BUILD_DIR) \
```